sphinx_gallery/gen_rst.py

```diff
@@ -1383,12 +1400,10 @@ def save_rst_example(
         + example_rst
     )
 
-    if time_elapsed >= gallery_conf["min_reported_time"]:
+    if time_elapsed > gallery_conf["min_reported_time"]:
```

Comment:
This will change the behavior for the default `min_reported_time = 0`... but maybe that's intentional and better behavior?

Reply:
Yes, the change was intentional; I thought it didn't make much sense to report the execution time for examples that were never executed.